src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -28,6 +29,18 @@ class AboutCommand extends ContainerAwareCommand
 {
     /**
      * {@inheritdoc}
+     *
+     * @deprecated since version 3.4, to be removed in 4.0
+     */
+    protected function getContainer()
```

Comment:
(could apply also to existing commands that alredy have the same deprecation btw?